db/migrate/20181126123517_configure_default_external_system.rb

```diff
@@ -0,0 +1,38 @@
+class ConfigureDefaultExternalSystem < ActiveRecord::Migration[5.2]
```

Comment:
This migration simply creates 2 ExternalSystem records - one for the primary atos queue which is in use at the moment and one for the secondary which is what we will use as we switch over to the new ATOS server